recipe/meta.yaml

```diff
@@ -8,53 +7,67 @@ package:
   version: {{ version|replace("-", "_") }}
 
 source:
-  fn: lokern_{{ version }}.tar.gz
   url:
-    - https://cran.r-project.org/src/contrib/lokern_{{ version }}.tar.gz
-    - https://cran.r-project.org/src/contrib/Archive/lokern/lokern_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/lokern_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/lokern/lokern_{{ version }}.tar.gz
   sha256: 3edb44213f0dca45047b91dd970e6aaf8b023f21855dd8ebf7d978b2563ababa
 
 build:
-  merge_build_host: true  # [win]
-  number: 0
-  skip: true  # [win32]
-
+  merge_build_host: True  # [win]
+  number: 1
   rpaths:
     - lib/R/lib/
     - lib/
+
 requirements:
   build:
-    - posix                # [win]
-    - {{ compiler('fortran') }}        # [unix]
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('fortran') }}        # [not win]
     - {{ compiler('m2w64_fortran') }}  # [win]
-    - {{ compiler('c') }}        # [unix]
-    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
   host:
     - r-base
     - r-sfsmisc >=1.0_12
-
   run:
     - r-base
+    - {{ native }}gcc-libs         # [win]
     - r-sfsmisc >=1.0_12
 
 test:
   commands:
-    - $R -e "library('lokern')"  # [not win]
+    - $R -e "library('lokern')"           # [not win]
     - "\"%R%\" -e \"library('lokern')\""  # [win]
 
 about:
   home: https://CRAN.R-project.org/package=lokern
   license: GPL-2.0-or-later
   summary: Kernel regression smoothing with adaptive local or global plug-in bandwidth selection.
-  license_family: GPL
-  license_file: {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```

Comment:
I'm not sure what to do here. The recipe originally bundled GPL-3. But now the cran skeleton wants to bundle GPL-2. Do we have strong feelings on how to interpret the CRAN license designation "GPL (>=2)"? I've seen conflicting examples, which makes me suspect that the behavior of the CRAN skeleton may have changed over time.

Reply:
answer: https://github.com/conda-forge/r-rtensor-feedstock/pull/8#discussion_r777388072